ts/utils/__tests__/dates.test.ts

```diff
@@ -35,13 +35,20 @@ describe("getExpireStatus", () => {
     expect(isExpired("2", "04")).toEqual(right(true));
   });
 
-  it("should mark the date as expired since we're passing the last month", () => {
-    const now = new Date(2020, 1, 1);
+  it("shouldn't mark the date as expired since we're in the last valid month", () => {
+    // 01/01/2020
+    MockDate.set(new Date(2020, 0, 1));
+    expect(isExpired(1, 2020)).toEqual(right(false));
+    // 31/01/2020
+    MockDate.set(new Date(2020, 0, 31));
+    expect(isExpired(1, 2020)).toEqual(right(false));
+  });
+
+  it("should mark the date as expired since we're passing the last valid month", () => {
```

Comment:
What do you mean by: "the last valid month"?

Reply:
You're right, the description isn't really clear. Updated in d42ac0b let me know if is more clear now!